src/core/Akka/Util/Helpers.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Akka.Util
+{
+    public static class Helpers
+    {
+        public static T Requiring<T>(this T obj, Func<T, bool> cond, string msg)
```

Comment:
This is a helper fluent build pattern function to do an "assert" in fluent manner.
Usage: `myConfig.GetInt("hocon.path.value").Requiring(i => i > 0, "value must be greater than zero.")`

Reply:
This is meant to be used in the main Akka.NET code base too then I take it? I.e. not just for testing?